src/Nest/Requests.Rollup.cs

```diff
@@ -106,7 +108,8 @@ Id Id
 		}
 	}
 
-	///<summary>Request for GetCapabilities <para></para></summary>
+	///<summary>Request for GetCapabilities</summary>
+	///<remarks>Note: Experimental within the Elasticsearch server, it is highly likely to break or be removed in the near future and has no backwards compatibility guarantees.</remarks>
```

Comment:
:heart:

Reply:
> it is highly likely to break or be removed in the near future

I'm not sure about this wording - is it highly likely? 

We could use the same wording as the reference docs:

> This functionality is experimental and may be changed or removed completely in a future release. Elastic will take a best effort approach to fix any issues, but experimental features are not subject to the support SLA of official GA features.